pandas/io/tests/test_pytables.py

```diff
@@ -237,6 +237,30 @@ def roundtrip(key, obj, **kwargs):
         finally:
             safe_remove(path)
 
+    def test_conv_categorical(self):
+
```

Comment:
move this right after ``test_categorical``, and rename to ``test_categorical_conversion``